management/connection.go

```diff
@@ -587,6 +590,25 @@ type ConnectionOptionsADFS struct {
 	SetUserAttributes *string `json:"set_user_root_attributes,omitempty"`
 }
 
```

Comment:
I'm not sure I got this right.

```suggestion

type ConnectionOptionsSAMLIdpinitiated struct {
	ClientID                          *string `json:"client_id,omitempty"`
	ClientProtocol                *string `json:"client_protocol,omitempty"`
	ClientAuthorizequery.    *string `json:"client_authorizequery"`
}

```